### DISCUSSION TITLE
Programming workflow

### DISCUSSION THREAD

**Discussion by josuah** [tinyvision-ai-inc/pico-ice#30] [Ideas]
For now, a port of https://github.com/adafruit/tinyuf2/ is in place.
- Does sound convenient for you to have an uf2 file to copy over to the FPGA drive?
- Would you expect something to flash using a single command to avoid manually mounting the drive an copying?
- For instance, would a batch and shell script for multiple platform to find the drive, mount it, and flash it... be convenient enough?
- Do you aim to have something like dfu-utils or OpenFGPALoader to work?
- What would be your general programming workflow with the FPGAs?

**Comment by vr2045** [comment]
Couple of use cases I can think of:
- GUI based drag/drop UF2 file into the USB drive for FPGA programming
- Makefile based copy of UF2 to USB drive
